src/components/index/FindCell.tsx

```diff
@@ -11,8 +12,10 @@ export const FindCell = () => {
                 <PageTitle title="Find Cell"/>
                 <div className="flex flex-col md:flex-row flex-wrap justify-between my-16">
                     <div className="w-full md:w-1/2 lg:w-2/5">
-                        <img
-                            src="https://images.unsplash.com/photo-1526746323784-6bc814d79273?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80"
+                        <Image
+                        width={1200}
+                        height={1200}
+                            src="https://res.cloudinary.com/dpbuspblx/image/upload/v1679958910/cla-cta/Home_cells_Facebook_b9pwvj.png"
                             className="w-full rounded-md" alt="Cell Group"/>
```

Comment:
can we set a fixed width height using tailwind (CSS) here as well. For some reason this section is expanding/shrinking depending on the dimensions of the image